nixos/modules/services/x11/desktop-managers/pantheon.nix

```diff
@@ -172,6 +172,17 @@ in
       programs.dconf.enable = true;
       networking.networkmanager.enable = mkDefault true;
 
+      systemd.services.touchegg = {
+        description = "Touchégg Daemonre";
+        wantedBy = [ "multi-user.target" ];
+
+        serviceConfig = {
+          Type = "simple";
+          ExecStart = "${pkgs.touchegg}/bin/touchegg --daemon";
+          Restart = "on-failure";
+        };
+      };
```

Comment:
Maybe add this as a separate module, I think this will be useful for applications like [touche](https://github.com/JoseExposito/touche).